### PR TITLE
Use CATS::DB::db

### PR DESCRIPTION
**Pull request by ptsolg**

### CODE REVIEW

File: cgi-bin/CATS/UI/ImportSources.pm [hunk 1]
```diff
@@ -55,14 +55,16 @@ sub download_frame {
...
-    local $dbh->{ib_enable_utf8} = 0;
-    my ($fname, $src) = $dbh->selectrow_array(q~
+    $CATS::DB::db->disable_utf8;
+    my @row = $dbh->selectrow_array(q~
         SELECT fname, src FROM problem_sources_local WHERE id = ? AND guid IS NOT NULL~, undef,
-        $p->{psid}) or return;
+        $p->{psid});
+    $CATS::DB::db->enable_utf8;
+    @row or return;
     $p->print_file(
```
Comment (klenin) [inline cgi-bin/CATS/UI/ImportSources.pm]: Why remove row unpacking?

File: cgi-bin/CATS/UI/Contests.pm [hunk 2]
```diff
@@ -29,7 +29,7 @@ sub contests_new_frame {
     init_template($p, 'contests_new.html.tt');
 
     my $date = $dbh->selectrow_array(qq~
-        SELECT CURRENT_TIMESTAMP $FROM_DUMMY~);
+        SELECT CURRENT_TIMESTAMP $db->{FROM_DUMMY}~);
```
Comment (klenin) [inline cgi-bin/CATS/UI/Contests.pm]: How did it work before without import?

Comment (ptsolg) [reply cgi-bin/CATS/UI/Contests.pm]: I removed '$FROM_DUMMY' and added '$db' in previous commit
https://github.com/klenin/cats-main/pull/350/commits/3be5a35f79ee66dceb63d3e021431da6edfe2b81#diff-f11fd1fbf18b894256063c6ee9eb8f14R12

File: cgi-bin/CATS/UI/ImportSources.pm (hunk 1 above)
Comment (ptsolg) [reply cgi-bin/CATS/UI/ImportSources.pm]: I thought that unpacking of empty array would cause an exception.

**Pull request merged**